Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/RectTransformCubeScaler.cs

```diff
@@ -17,6 +17,10 @@ public class RectTransformCubeScaler : MonoBehaviour
         private RectTransform rectTransform;
         private Vector2 prevRectSize = Vector2.zero;
 
+        [SerializeField]
+        private Transform objectToScale = null;
+        private Transform prevObjectToScale = null;
+
         private void Start()
         {
             rectTransform = GetComponent<RectTransform>();
```

Comment:
does it make sense to have the gameobject this component is attached to set up as default if the objectToScale = null?

Reply:
I thought about that, but decided against it.  Doing that is what caused the in-editor clickable region to be so large in the first place, so I don't think it's a good default behavior.